typescript/cli/src/rebalancer/executor/Executor.ts

```diff
@@ -19,10 +20,10 @@ import { type BridgeConfig, getBridgeConfig } from '../utils/bridgeConfig.js';
 export class Executor implements IExecutor {
   constructor(
     private readonly bridges: ChainMap<BridgeConfig>,
-    private readonly rebalancerKey: string,
     private readonly warpCore: WarpCore,
     private readonly chainMetadata: ChainMap<ChainMetadata>,
     private readonly tokensByChainName: ChainMap<Token>,
+    private readonly context: WriteCommandContext,
```

Comment:
Instead of the context, just receive the multiprovider given that it is the only thing used from it.